src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -146,6 +146,7 @@
   {:fx [[:dispatch
          [:standard-auth/authorize-with-password
           {:blur?             true
+           :skip-biometric?   true
```

Comment:
this a little bit confusing, it says with password but skip-biometric? true is needed

Reply:
Indeed, I think a better name would be `:hide-biometrics-button?`. WDYT?